R/profilePlot.R

```diff
@@ -2,102 +2,104 @@
 #'
 #' A profile plot shows Log Intensity on the X axis and Log Ratio on the Y axis.
 #' This function is intended to show the profile plot from a dataframe
-#' created by topTable or topTreat.  Properly normalized data will generally be
+#' in DGEobj design objects (i.e BDL_vs_SHAM).  Properly normalized data will generally be
 #' centered around LogRatio = 0.
 #'
 #' By default, the plot places "logFC" on the Y axis and "AveExpr" on the X
 #' axis. By default, a reference horizontal line is shown at LogRatio = 0 on the
 #' Y axis. Optionally, additional reference lines will be drawn at +/- a user
 #' supplied LogRatio threshold. A Loess line fit is drawn through the actual
-#' data. The points are color coded using the significance measure (p-value or
-#' FDR threshold) supplied by the user. By default, the P.Value field is used
+#' data. The points are color coded using the significance measure (i.e p-value)
+#' supplied by the user. By default, the P.Value field is used
 #' with a threshold of 0.01 to color code the points.
 #'
+#' The profilePlot has also been implemented
```

Comment:
We dont have baseFont and theme attributes. This section needs to be removed.

Reply:
done